### DISCUSSION TITLE
analog reading in cm

### DISCUSSION THREAD

**Discussion by unbreakmat** [jomjol/AI-on-the-edge-device#3264] [General]
### Issue

Hello i have a meter in cm for my fuel tank (from 0 to 150cm name: EURO-INDEX MT-Profil ) is there a way that ai on the edge can result those image correctly? i will do some calculations with home assistant to calculate the volume of my tank
ive tried it with the software and i get wrong values like 9 or 9,2 , can someone help me?

**Comment by caco3** [comment]
You would need to add some post processing. Our firmware expects a pointer to go 360°. Thus the detected angle gets converted to a value of 0..1.
